homeassistant/const.py

```diff
@@ -752,11 +752,6 @@ class Platform(StrEnum):
 ENTITY_CATEGORY_CONFIG: Final = "config"
 ENTITY_CATEGORY_DIAGNOSTIC: Final = "diagnostic"
 ENTITY_CATEGORY_SYSTEM: Final = "system"
-ENTITY_CATEGORIES: Final[list[str]] = [
```

Comment:
Yeah, but we can't remove this, because it will break custom integrations...

Reply:
We can still introduce a new place / method, but these old ones need to stay here for a year or so 😬

Reply:
Putting it in `entity.py` didn't work out since I had to import it into `service.py` which is a circular import

Reply:
Is it bad practise to use a late import inside `service.py`?
```python
    # pylint: disable=import-outside-top-level
    from .entity import EntityCategory

    for ent_entry in ent_reg.entities.values():
        # Do not add config or diagnostic entities referenced by areas or devices

        if ent_entry.entity_category in EntityCategory:
            continue
```